homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -268,6 +269,12 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
     ATTR_PURIFY_VOLUME,
     ATTR_TEMPERATURE,
 )
+PURIFIER_3C_SENSORS = (
+    ATTR_FILTER_LIFE_REMAINING,
+    ATTR_FILTER_USE,
+    ATTR_MOTOR_SPEED,
+    ATTR_PM25,
```

Comment:
Maybe add pm25 device class to this entity description?

Reply:
Done